plugins/dbnd-docker/src/dbnd_docker/kubernetes/kubernetes_engine_config.py

```diff
@@ -92,15 +116,16 @@ class KubernetesEngineConfig(ContainerEngineConfig):
     requests = parameter.none()[Dict]
     limits = parameter.none()[Dict]
 
-    pod_exit_code_to_retry_count = parameter(empty_default=True).help(
-        "Mapping between pod exit code to amount of pod retry attempts"
+    pod_error_cfg_source_dict = parameter().help(
+        "Configurable values for pod error handling configuration"
     )[Dict]
-    pod_retry_delay = parameter.help(
-        "The delay between each pod retry attempt in time delta format. 1m, 5s, 1h, etc."
+    pod_error_cfg = parameter(
```

Comment:
it will not work - the value is calculated during the run , not during compile time     PodRetryConfiguration(pod_error_cfg_source_dict) -> PodRetryConfiguration(ParameterFactory)

Reply:
yes, fixd

Reply:
yes, fixd